base_layer/wallet/src/output_manager_service/storage/sqlite_db.rs

```diff
@@ -622,6 +625,10 @@ impl TryFrom<OutputSql> for UnblindedOutput {
                     .ok_or_else(|| OutputManagerStorageError::ConversionError)?,
                 maturity: o.maturity as u64,
             },
+        };
+        Ok(Self {
+            unblinded_output,
+            hash: Vec::new(),
```

Comment:
So this should come from the DB? Just haven't done the migration yet?

Reply:
This is only temp for this wip PR, this will come from the db.